packages/editor-ui/src/modules/credentials.ts

```diff
@@ -65,8 +68,12 @@ const module: Module<ICredentialsState, IRootState> = {
 		enableOAuthCredential(state: ICredentialsState, credential: ICredentialsResponse) {
 			// enable oauth event to track change between modals
 		},
+		...credentialsEEModule.mutations,
```

Comment:
maybe we can create a general utility method for extending these modules
at the end of file, can be something like
`export default extendModule(module, credentialsEEModule);`

Reply:
Good point! Added a generic typed utility function to extend Vuex modules.